### ISSUE TITLE
Incorrect iOS detection with Chrome on multi-touch supported Windows device (e.g. Surface Go)

### ISSUE THREAD

**Issue by fukayatsu** [codemirror/CodeMirror#6532]
After I updated CM to 5.59.0, `Ctrl+V` does not paste clipboard contents on Chrome on Surface Go.

This commit seems to be the cause: https://github.com/codemirror/CodeMirror/commit/7f3c36619f964d20e20c0ff5bec9cee99dae1549 

```
> userAgent = navigator.userAgent
"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36"

> platform = navigator.platform
"Win32"

> edge = /Edge\/(\d+)/.exec(userAgent)
null

> navigator.maxTouchPoints
10

> ios = !edge && /AppleWebKit/.test(userAgent) && (/Mobile\/\w+/.test(userAgent) || navigator.maxTouchPoints > 2)
true
```

**Comment by marijnh** [comment]
Ugh. Vendors seem really dedicated to sabotaging platform detection lately. I don't have an up-to-date iPad to test on—could you see if attached patch helps here?

**Comment by fukayatsu** [comment]
Sure!

I'll try it on iPad OS 14.3.

**Comment by fukayatsu** [comment]
e5b87f6 works fine on iPad and Surface Go 🎉 

---

- [x] Safari mobile mode on iPadPro (iPad OS 14.3)
    - UA: `Mozilla/5.0 (iPad; CPU OS 14_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0.2 Mobile/15E148 Safari/604.1`
    - safari: true
    - ios: true
- [x] Safari desktop mode on iPadPro (iPad OS 14.3)
    - UA: `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0.2 Safari/605.1.15`
    - safari: true
    - ios: true
- [x] Chrome on Surface Go (Windows 10)
    - UA: `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36`
    - safari: false
    - ios: false

**Comment by marijnh** [comment]
Thanks! I'll apply the same patch to my other projects then.

**Comment by marijnh** [comment]
Since this was causing quite some problems I've released a 5.59.1 emergency fix.
